### DISCUSSION TITLE
Make Auto play in the main menu when Storyboard background is enabled

### DISCUSSION THREAD

**Discussion by JuLi0n21** [ppy/osu#22426] [Ideas]
Many Storyboards are mapped to a certian map, so it would be nice if auto would also play in the background to make the sotryboard feel less empty

A toggle to show auto play when storybaord is enabled or not.

Also making the osu cookie see through would be nice when maximized

**Comment by peppy** [comment]
I feel like this could be a toggle or a different main menu background dropdown option (Storyboard + Gameplay). Interesting idea.

**Comment by Gabixel** [comment]
I would like to throw some ideas here if they can be interesting.

I understand that for such a "small" feature it would not be fair to spend a lot of time on it, but this suggestion can lead to various small neat ideas (and maybe even easter eggs). This could also enrich the main menu with some style.

I can imagine the possibility of:
- **randomly pulling a** local **replay**, if any, or even fetching a replay of, say, the top 10 in country/global (without going into detail about how);
- **manually starting to click** the circles / the objects (depending on the ruleset), so that the autoplay/replay will disappear and it lets you continue the map as a pastime (and vice-versa if you go idle the replay returns);
- allowing the **smoke** so that you can draw in the main menu (not sure if this can be done for the other rulesets);
- using autoplay but sometimes with **random mod combinations and customizations** (maybe excluding some very weird ones if they can generate issues). I might as well accept excluding rate-changing mods if it can interfere with how audio should work in the menu;
- still using autoplay, but with a future implementation of **cursor dancing** (maybe like a rare chance of this to happen or by using a hotkey)?

**Reply by Kuina-chan** [reply]
This actually a neat idea to enrich the experience for players!
